tsconfig.json

```diff
@@ -11,10 +11,12 @@
     "esModuleInterop": true,
     "noImplicitAny": true,
     "resolveJsonModule": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "incremental": true,
```

Comment:
Great 👍 We can now remove `incremental` flag from package.son: (`tsc -p tsconfig-build.json --incremental`)

Reply:
Done in db15bcb